src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java

```diff
@@ -91,21 +88,22 @@ static void parse(byte[] b, int len, Map<String, Object> data, boolean saveAnnot
                     break;
                 }
                 buf.append(line, pos, preamble);
-                annotations.add(ImmutableMap.of(POSITION_KEY, buf.length(), NOTE_KEY, line.substring(endOfPreamble, postamble)));
+                annotations.add(ImmutableMap.of(POSITION_KEY, buf.length(), NOTE_KEY,
+                        line.substring(endOfPreamble, postamble)));
                 pos = postamble + ConsoleNote.POSTAMBLE_STR.length();
             }
             buf.append(line, pos, line.length()); // append tail
             data.put(MESSAGE_KEY, buf.toString());
-            if (saveAnnotations)
-            {
-              data.put(ANNOTATIONS_KEY, annotations);
+            if (saveAnnotations) {
+                data.put(ANNOTATIONS_KEY, annotations);
             }
         }
     }
 
     static void write(Writer w, Map<String, Object> source) throws IOException {
-        if(source == null) throw new NullPointerException("source is null");
-        String message = (String)source.get(MESSAGE_KEY);
+        if (source == null)
+            throw new NullPointerException("source is null");
```

Comment:
This should also be changed. Either single line or curly braces would be my recommendation.